src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -0,0 +1,161 @@
+package org.broadinstitute.hellbender.utils.fermi;
```

Comment:
Eh, if it's a huge pain you can leave it.  It's not a big deal either way.  It just seems a bit weird.  Naming is always hard.